Makefile

```diff
@@ -58,9 +58,9 @@ manifests: controller-gen ## 🪄 Generate k8s manifests
 		output:rbac:dir=deploy/kustomize/rbac \
 		output:webhook:dir=deploy/kustomize/webhook \
 		output:crd:dir=deploy/kustomize/crd/bases
-	echo '{{- if .Values.installCRDs }}' > deploy/charts/rig-operator/templates/crd.yaml
-	cat 'deploy/kustomize/crd/bases/rig.dev_capsules.yaml' >> deploy/charts/rig-operator/templates/crd.yaml
-	echo '{{- end }}' >> deploy/charts/rig-operator/templates/crd.yaml
+	# echo '{{- if .Values.installCRDs }}' > deploy/charts/rig-operator/templates/crd.yaml
+	# cat 'deploy/kustomize/crd/bases/rig.dev_capsules.yaml' >> deploy/charts/rig-operator/templates/crd.yaml
+	# echo '{{- end }}' >> deploy/charts/rig-operator/templates/crd.yaml
```

Comment:
Do we have an alternative to this for generating CRD spec?

Reply:
we are already generating the CRD spec, this is generating a helm template based on the yaml which `controller-gen` outputs. We could maybe do something else, but templates for templates quickly get confusing xD